wallet/storage/src/internal/test.rs

```diff
@@ -35,36 +27,3 @@ fn storage_get_default_version_in_tx() {
         assert_eq!(vtx, vst, "Transaction and non-transaction inconsistency");
     })
 }
-
-#[cfg(not(loom))]
-#[rstest]
-#[trace]
-#[case(Seed::from_entropy())]
-fn read_write_utxo_in_db_transaction(#[case] seed: Seed) {
```

Comment:
Why are we deleting this test?

Reply:
Because it was testing UTXO DB store/load which was removed.